### DISCUSSION TITLE
Option to choose Mica effect

### DISCUSSION THREAD

**Discussion by amrbashir** [zen-browser/desktop#3482] [Ideas]
There is at least two mica effects that exist in Windows 11:
1. Mica
2. Mica Alt

Can be seen in this image 
![99811d26-be07-40a9-8451-f953665f8aef-image](https://github.com/user-attachments/assets/97c5de3c-07b3-41ef-b3e0-5e3c5c198ec1)

I believe Zen is using Mica Alt at the moment but I find it a bit annoying for me to adapt to and I would like the option to use the normal Mica effect.

Edit: A workaround is to use [MicaForEveryone](https://github.com/MicaForEveryone/MicaForEveryone) to enforce this.

**Comment by weber-man** [comment]
All the Windows Apps use Mica not Mica Alt as their backdrops. I think Mica should be used as default here.
